pubkey_encrypt.info.yml

```diff
@@ -1,5 +1,5 @@
 name: Pubkey Encrypt
-description: Adds Credentials-based Public Key Encryption support.
+description: Provides support for encrypting data with users login-credentials.
```

Comment:
Same issue with "users" here too.  Also, I don't think "login-credentials" should be hyphenated here and in various places in the README.  Can probably just remove the "-" everywhere.